website/public/demo/index.ejs.html

```diff
@@ -121,14 +133,13 @@ <h2>Demo</h2>
       </p>
 
       <pre id="shadow-code-input"></pre>
-      <textarea id="code-input" spellcheck="false">
+<div contenteditable="true" id="code-input" spellcheck="false">
```

Comment:
Blocking: contenteditable (and maybe some other things) seems to break a lot of things.

For example, if I clear the text box, then type 'a', the cursor stays *before* the 'a'. It should be *after* the 'a'. If I then type 's' then 'd' then 'f', the text box shows "fdsa" (instead of "asdf").

Weird things also happen when I type some digits then press the enter key a few times.

I'm pretty sure these issues are because of markEditorText. When I originally made the demo, I tried making contenteditable work, but there were too many issues to fight. For example, even after fixing the issues I mentioned above, undo-redo was broken.

Let's keep this as `<textarea>`, unless you can make the contenteditable approach high-quality. I don't think making contenteditable work is possible, though; even Gmail's email composer is full of bugs *in plain text mode*.